accounting/formatters/BaseFormatter.py

```diff
@@ -34,7 +34,8 @@ def break_chars(s):
         ],
     "table": [
         "font-size: 10pt",
-        "border: 1px solid black",
+        "border-collapse: collapse",
+	"border-color: #ffffff",
```

Comment:
It looks like your editor uses tabs instead of spaces for indenting. For consistency, let's keep to using spaces.
```suggestion
        "border-color: #ffffff",
```